src/test/regress/expected/subselect_gp.out

```diff
@@ -3112,3 +3112,201 @@ select * from r where b in (select b from s where c=10 order by c limit 2);
  1 | 2 | 3
 (1 row)
 
+-- Test that Explicit Redistribute Motion is applied properly for
```

Comment:
You should modify `/src/test/regress/expected/subselect_gp_1.out` too. See that file commit history for details. You didn't face CI issues as far as ppc64le tests are muted now.

Reply:
done